src/libAtomVM/atom.h

```diff
@@ -61,7 +65,7 @@ int atom_are_equals(AtomString a, AtomString b);
  */
 static inline int atom_string_len(AtomString atom_str)
 {
-    return *((const uint8_t *) atom_str);
+    return *((const uint8_t *)atom_str);
```

Comment:
we are not using this code style, I think that I should clarify it in our CONTRIBUTING file or setup a formatter for C

Reply:
I guess a standardized C formatter would be better.
Nevertheless, what do you mean with "this code style"?